desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -751,7 +751,7 @@ protected boolean onWindowClosing() {
 
                 if (unsavedChangesChoice == 0) { // save changes
                     _saveButton.doClick();
-                } else if (unsavedChangesChoice == 2) { // cancel closing
+                } else if (unsavedChangesChoice != 1) { // cancel closing
```

Comment:
This looks right, but could we switch to using the symbolic constants from JOptionPane instead of the int values here? I know this is nothing changed in your PR, but since you're doing this improvement we might as well improve the code readabiity by comparing with ```JOptionPane.WHATEVER``` instead of 0 and 1